harness/determined/exec/gc_checkpoints.py

```diff
@@ -24,7 +24,10 @@ def delete_checkpoints(
     for storage_id in to_delete:
         if not dry_run:
             logging.info(f"Deleting checkpoint {storage_id}")
-            manager.delete(storage_id)
+            try:
+                manager.delete(storage_id)
+            except errors.CheckpointNotFound as e:
+                logging.warn(str(e))
```

Comment:
Why str()?

Reply:
I guess I normally use it within a f expression string and it is needed in that case.  
Here I perhaps not since it knows it wants a str?

Reply:
Shall I take it out?

Reply:
`logging.warn` interally will call `str` on the passed msg.

Reply:
at least, idk, thought this gotta called to fmt and print the record generated by `logging.warning` https://github.com/python/cpython/blob/main/Lib/logging/__init__.py#L383-L393

Reply:
doesn't hurt anything either why